internal/handlers/images.go

```diff
@@ -59,12 +63,31 @@ func NewImageHandler(is imagestore.ImageStore, assistedServiceScheme, assistedSe
 		Recorder: metrics.NewRecorder(metricsConfig),
 	})
 
+	client := http.DefaultClient
+	if caCertFile != "" {
+		caCert, err := ioutil.ReadFile(caCertFile)
+		if err != nil {
+			log.Fatalf("Error opening cert file %s, %s", caCertFile, err)
+		}
+		caCertPool := x509.NewCertPool()
+		caCertPool.AppendCertsFromPEM(caCert)
+
+		t := &http.Transport{
+			TLSClientConfig: &tls.Config{
+				RootCAs:    caCertPool,
+				MinVersion: tls.VersionTLS12,
```

Comment:
What kind of errors were you seeing that prompted specifying a minimum version?

Reply:
The linter was not happy with me. This was the minimum version that made it stop complaining.

Reply:
Fair enough